presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -403,6 +402,12 @@ private synchronized void updateTaskInfo(TaskInfo newValue, List<TaskSource> sou
         partitionedSplitCountTracker.setPartitionedSplitCount(getPartitionedSplitCount());
     }
 
+    private void updateTaskInfo(TaskInfo taskInfo)
```

Comment:
I like the name change from `updateTaskInfo` to `processTaskUpdate`, but I don't understand why this method was extracted.  It doesn't seem to be used anywhere except `processTaskUpdate`.